JSTests/test262/expectations.yaml

```diff
@@ -2026,18 +1897,9 @@ test/language/identifiers/start-unicode-13.0.0-class.js:
 test/language/identifiers/start-unicode-14.0.0-class.js:
   default: "SyntaxError: Invalid character: '#'"
   strict mode: "SyntaxError: Invalid character: '#'"
-test/language/identifiers/start-unicode-15.0.0-class-escaped.js:
-  default: "SyntaxError: Invalid unicode escape in identifier: '#\\u{1123F}'"
-  strict mode: "SyntaxError: Invalid unicode escape in identifier: '#\\u{1123F}'"
 test/language/identifiers/start-unicode-15.0.0-class.js:
   default: "SyntaxError: Invalid character: '#'"
   strict mode: "SyntaxError: Invalid character: '#'"
-test/language/identifiers/start-unicode-15.0.0-escaped.js:
-  default: "SyntaxError: Invalid unicode escape in identifier: '\\u{1123F}'"
-  strict mode: "SyntaxError: Invalid unicode escape in identifier: '\\u{1123F}'"
-test/language/identifiers/start-unicode-15.0.0.js:
-  default: "SyntaxError: Invalid character '\\ud804'"
-  strict mode: "SyntaxError: Invalid character '\\ud804'"
```

Comment:
I don't think those are related by those changes either. Likely something gets fixed previously in the repo but test262 haven't been run/added to the commit.